src/components/Copy/copy.scss

```diff
@@ -0,0 +1,31 @@
+@import "../../variables.scss";
+
+.copy {
+  cursor: pointer;
+
+  &__container {
+    position: relative;
+  }
+
+  &__tooltip {
+    position: absolute;
+    font-size: .75rem;
+    box-sizing: border-box;
+    padding: .5rem;
+    width: 56px;
+    left: 50%;
+    margin-left: -28px;
```

Comment:
this tooltip implementation doesn't seem extendable. How does this show up on mobile?